content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,187 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires and more!
+descriptionLong: |
+  Monthly updates are here! Read all about what our community is doing
+  with DVC, our CEO's interviews and talks, new team members and more!
+picture: 2021-04-15/april21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718
+tags:
+  - Heartbeat
+  - DVC
+  - MLOps
+  - Made with ML
+  - AI Singapore
+  - DAGsHub
+---
+
+## From the Community
+
+We're starting with the community this month because it has been overflowing
+with great content from our users. It's like we're on a sugar high!
+
+![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)
+
+### Goku Mohandas' New Lessons!
+
+First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of
+[Made With ML](https://madewithml.com/) has created
+[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.  
+It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```

Comment:
Per https://github.com/iterative/dvc.org/pull/2372#discussion_r613693094:

```suggestion
[Made With ML](https://madewithml.com/) has added this
[Versioning](https://madewithml.com/courses/mlops/versioning/) lesson to the
MLOps course using DVC. It's RT'ing around the ML Twitter space like hotcakes!
🥞
```

Reply:
p.s. please retweet some hotcakes my way.

Reply:
Much better.  Thank you!  I'm correcting this way on my side and not committing suggestion because that's how I usually get in to trouble.  Trying to get this out the door..... Maybe that makes this more complicated?